source/Dgraph.tests.e2e/Tests/Approved/SchemaTest.AlterSchemAsExpected.approved.txt

```diff
@@ -1,5 +1,8 @@
 abool: bool .
+dgraph.drop.op: string .
+dgraph.graphql.p_query: string @index(sha256) .
 dgraph.graphql.schema: string .
+dgraph.graphql.xid: string @index(exact) @upsert .
```

Comment:
Have you tested push this schema? as far as I remember we can't use any predicate with prefix "dgraph.*".